src/main.cpp

```diff
@@ -163,6 +163,10 @@ void setupNetwork() {
     BleFingerprintCollection::forgetMs = AsyncWiFiSettings.integer("forget_ms", 0, 3000000, DEFAULT_FORGET_MS, "Forget beacon if not seen for (in milliseconds)");
     BleFingerprintCollection::txRefRssi = AsyncWiFiSettings.integer("tx_ref_rssi", -100, 100, DEFAULT_TX_REF_RSSI, "Rssi expected from this tx power at 1m (used for node iBeacon)");
 
+    AsyncWiFiSettings.heading("MiFlora <a href='https://github.com/0Flush0/ESPresenseFlora' target='_blank'>ℹ️</a> ", false);
```

Comment:
Since you filter on the fingerprint why is the mac filter needed?

Reply:
When you only want to query specific macs for each room, what do you do then? Is this allready implemented?

Reply:
Oh, that makes some sense, but i'd think using the query filter would do this nicely, right?  Just put in flora:{mac} in query on just that node.

Reply:
True, i have removed that, this feature also caused longer website loading time for me.